nshmdb/fault.py

```diff
@@ -50,7 +49,7 @@ class FaultPlane:
         The rake angle of the fault plane.
     """
 
-    _corners: np.ndarray
+    corners_nztm: np.ndarray
```

Comment:
Needs to be added to the docstring, also is there a specific order to the corners? Specifically regarding which two corners define the top-edge line?

Reply:
Will update, and yes there is, I'll document that in both the `corners` property and the `corners_nztm` variable.